libs/ui-lib/lib/cim/components/ClusterDeployment/use-networking-formik.ts

```
@@ -44,16 +47,33 @@ const getInfraEnvProxy = (infraEnvs: InfraEnvK8sResource[]) => {
   initialValues: ClusterDeploymentNetworkingValues,
   hostSubnets: HostSubnets,
+  openshiftVersion?: string,
 ) =>
   Yup.lazy((values: ClusterDeploymentNetworkingValues) =>
     Yup.object<ClusterDeploymentNetworkingValues>().shape({
-      clusterNetworkHostPrefix: hostPrefixValidationSchema(values.clusterNetworkCidr),
-      clusterNetworkCidr: ipBlockValidationSchema(values.serviceNetworkCidr),
-      serviceNetworkCidr: ipBlockValidationSchema(values.clusterNetworkCidr),
+      clusterNetworks: clusterNetworksValidationSchema.when('stackType', {
+        is: (stackType: NetworkConfigurationValues['stackType']) => stackType === IPV4_STACK,
+        then: () => clusterNetworksValidationSchema.concat(IPv4ValidationSchema),
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)